### ISSUE TITLE
Doesn't work with Rails 5?

### ISSUE THREAD

**Issue by tjchecketts** [vinsol-spree-contrib/spree-on-react#19]
Does this not work with Rails 5? Got this error when running 'bundle' in the terminal:

    Bundler could not find compatible versions for gem "activesupport":
    In snapshot (Gemfile.lock):
    activesupport (= 5.1.4)

    In Gemfile:
    web-console (>= 3.3.0) was resolved to 3.5.1, which depends on
    actionview (>= 5.0) was resolved to 5.1.4, which depends on
    activesupport (= 5.1.4)

    rails (~> 4.2.7) was resolved to 4.2.7, which depends on
    activesupport (= 4.2.7)

Is there any troubleshooting or methods to make this work with the newest Rails?

**Comment by ShubhamGupta** [comment]
This doesn't seem to be related to this project. Can you still share your Gemfile for further debugging.

**Comment by tjchecketts** [comment]
Posted the wrong error message, sorry! This is the original error message I got, so I tried bumping back the rails version and then got the error message that I already sent:
```
Bundler could not find compatible versions for gem "rails":
  In snapshot (Gemfile.lock):
    rails (= 5.1.4)

  In Gemfile:
    rails (~> 5.1.4)

    spree (~> 3.1.0) was resolved to 3.1.0, which depends on
      spree_core (= 3.1.0) was resolved to 3.1.0, which depends on
        rails (~> 4.2.6)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```
When I run `bundle update` it gives me the same error.

Here's my gemfile:
```
source 'https://rubygems.org'

git_source(:github) do |repo_name|
  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
  "https://github.com/#{repo_name}.git"
end

gem 'spree', '~> 3.1.0'
gem 'spree_auth_devise', '~> 3.1.0'
gem 'spree_ams', github: 'vinsol-spree-contrib/spree_ams', branch: '3-1-stable'
gem 'rack-cors', require: 'rack/cors'

# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
gem 'rails', '~> 5.1.4'
# Use postgresql as the database for Active Record
gem 'pg', '~> 0.18'
# Use Puma as the app server
gem 'puma', '~> 3.7'
# Use SCSS for stylesheets
gem 'sass-rails', '~> 5.0'
# Use Uglifier as compressor for JavaScript assets
gem 'uglifier', '>= 1.3.0'
# See https://github.com/rails/execjs#readme for more supported runtimes
# gem 'therubyracer', platforms: :ruby

# Use CoffeeScript for .coffee assets and views
gem 'coffee-rails', '~> 4.2'
# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
gem 'turbolinks', '~> 5'
# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
gem 'jbuilder', '~> 2.5'
# Use Redis adapter to run Action Cable in production
# gem 'redis', '~> 3.0'
# Use ActiveModel has_secure_password
# gem 'bcrypt', '~> 3.1.7'

# Use Capistrano for deployment
# gem 'capistrano-rails', group: :development

group :development, :test do
  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
  # Adds support for Capybara system testing and selenium driver
  gem 'capybara', '~> 2.13'
  gem 'selenium-webdriver'
end

group :development do
  # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
  gem 'web-console', '>= 3.3.0'
  gem 'listen', '>= 3.0.5', '< 3.2'
  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
  gem 'spring'
  gem 'spring-watcher-listen', '~> 2.0.0'
end

# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```

**Comment by ShubhamGupta** [comment]
Issue not linked with this project, Closing.
